doc/source/changelog.rst

```diff
@@ -5,6 +5,7 @@ Changelog
 ------
 
 -  Handle category dtype
+-  Fix download of reports
```

Comment:
Is this from a previous commit that we forgot ?

Reply:
Indeed. Maybe we should remove it ?